packages/e2e-tests/plugins/inner-blocks-renderappender.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Plugin Name: Gutenberg Test InnerBlocks renderAppender
+ * Plugin URI: https://github.com/WordPress/gutenberg
+ * Author: Gutenberg Team
+ *
+ * @package gutenberg-test-inner-blocks-renderappender
+ */
+
+/**
+ * Registers a custom script for the plugin.
+ */
+function enqueue_inner_blocks_renderappender_script() {
```

Comment:
By its camelCase `renderAppender`, we should treat "render appender" as two words, reflecting this as well in its `kebab-case` and `snake_case` forms as `render-appender.php` and `render_appender` respectively.